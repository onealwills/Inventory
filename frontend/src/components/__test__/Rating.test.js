import { render } from "@testing-library/react";
import Rating from "../Rating";

const rating = {
  rating: 3.5,
  numReviews: 5,
  caption: "whatever",
};

test("<Rating/> component Snapshot", () => {
  const component = render(<Rating rating={rating} caption=""></Rating>);
  expect(component.container).toMatchSnapshot();
});
