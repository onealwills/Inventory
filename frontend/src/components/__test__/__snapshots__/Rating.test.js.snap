// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Rating/> component Snapshot 1`] = `
<div>
  <div
    class="rating"
  >
    <span>
      <i
        class="fa fa-star-o"
      />
    </span>
    <span>
      <i
        class="fa fa-star-o"
      />
    </span>
    <span>
      <i
        class="fa fa-star-o"
      />
    </span>
    <span>
      <i
        class="fa fa-star-o"
      />
    </span>
    <span>
      <i
        class="fa fa-star-o"
      />
    </span>
    <span>
      undefined reviews
    </span>
  </div>
</div>
`;
